actnow/profiles/views.py

```diff
@@ -18,7 +18,7 @@ class UserProfileView(ModelViewSet):
 
     def get_permissions(self):
         if self.action == "create":
-            permission_classes = [IsAuthenticated]
+            permission_classes = [DenyAll]
```

Comment:
Rather than use DenyAll, we can just declare `UserProfileView` without [`mixins.CreateModelMixin`](https://github.com/encode/django-rest-framework/blob/98e56e0327596db352b35fa3b3dc8355dc9bd030/rest_framework/viewsets.py#L235)